qa/common/gen_qa_models.py

```diff
@@ -2854,6 +2936,112 @@ def create_fixed_models(
             32,
         )
 
+    if FLAGS.tensorrt_bf16:
+        if os.getenv("GPU_MODEL") == "A100":
+            # Dynamic shape
+            create_plan_modelconfig(
+                FLAGS.models_dir,
+                8,
+                1,
+                [-1, -1],
+                [-1, -1],
+                [-1, -1],
+                trt.bfloat16,
```

Comment:
Let's create a dummy class called np_dtype_bfloat16 and document there saying as numpy does not support BF16 datatype, we would use this as a representative. 
Internally, we can handle this object to convert to trt.bfloat16 for plan models.

Reply:
Let's not expose trt abstraction here and try to keep it generic to add support for bf16 in other backends as well.